app/main/views.py

```diff
@@ -70,6 +72,32 @@ def update_editor_contents():
 
     return 'OK', 200
 
-@main.route('/resource-view')
+@csrf.exempt
+@main.route('/resource-view', methods =['POST'])
+def post_rating():
+
+    if request is not None:
+            print request.json
+            print request.form
+            print "im here!"
+            time = datetime.now()
+            star_rating = request.json['rating']
+            comment = request.json['review']
+            if comment and star_rating:
+                rating = Rating(submission_time = time,
```

Comment:
nit: no space between variable and value for all Rating() creations in this view
`submission_time=time`